cmd/get/clusters/printer_test.go

```diff
@@ -248,33 +248,33 @@ func Test_printOutput(t *testing.T) {
 	}
 }
 
-func newCAPIV1alpha3Cluster(id, created, release, org, description string, conditions []string) *capiv1alpha3.Cluster {
+func newcapiv1beta1Cluster(id, created, release, org, description string, conditions []string) *capiv1beta1.Cluster {
 	location, _ := time.LoadLocation("UTC")
 	parsedCreationDate, _ := time.ParseInLocation(time.RFC3339, created, location)
-	c := &capiv1alpha3.Cluster{
+	c := &capiv1beta1.Cluster{
 		TypeMeta: metav1.TypeMeta{
```

Comment:
is the `TypeMeta` field required? I don't think I've ever set it, and it requires being changed every time we upgrade the api version. Maybe we could remove it?

Reply:
I'm alright with removing it but I can't tell you if it's going to break anything.

Reply:
In this test (and a couple others) we seem to need it and it breaks when removed. However, for the `template` commands it was fine to remove it and I did.
As for the `get` commands, I was under the impression that we set it explicitly to ensure the CRs will be returned in a particular API version.

Reply:
ok, let's keep it for now then